manifests/m/Microsoft/Bicep/0.6.1/Microsoft.Bicep.locale.en-US.yaml

```
@@ -0,0 +1,27 @@
+# Created using wingetcreate 1.0.4.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.1.0.schema.json
+
+PackageIdentifier: Microsoft.Bicep
+PackageVersion: 0.6.1

```

```suggestion
PackageVersion: 0.6.1.6515
```
This must match Control Panel.

You will also need to change the version folder from `0.6.1` to `0.6.1.6515`